core/domain/exp_jobs_one_off.py

```diff
@@ -51,13 +54,39 @@ def entity_classes_to_map_over(cls):
     @staticmethod
     def map(exploration_model):
         if not exploration_model.deleted:
-            exp_services.create_exploration_summary(exploration_model.id)
+            exp_services.create_exploration_summary(
+                exploration_model.id, contributor_id_to_add=None)
```

Comment:
In all of these calls, do not specify contributor_id_to_add, since it no longer has a default value. The call should be

            exp_services.create_exploration_summary(exploration_model.id, None)